scripts/cc-cli/src/test/integration-tests/collectCoins.test.ts

```diff
@@ -13,11 +13,9 @@ import {
 } from "creditcoin-js";
 import { cryptoWaitReady } from "@polkadot/util-crypto";
 import { deployCtcContract, CREDO_PER_CTC } from "creditcoin-js/lib/ctc-deploy";
-import {
-  forElapsedBlocks,
-  testData,
-  tryRegisterAddress,
-} from "creditcoin-js/lib/testUtils";
+import { testData, tryRegisterAddress } from "creditcoin-js/lib/testUtils";
+
+import { forElapsedBlocks } from "creditcoin-js/lib/testUtils/index";
```

Comment:
This change shouldn't be necessary.

Reply:
This was causing build issues for some reason. Even after rebuilding creditcoin-js and updating the metadata. Still looking for a reason but I agree it shouldn't be necessary

Reply:
could it be that you have a stale yarn cache? Try `rm ~/.cache/yarn` before reinstalling creditcoin-js.

Reply:
For some reason, there was an extra `testUtils.js` in the build cache of `creditcoin-js` that was overshadowing the correct module. After removing that it works as expected.